pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -986,9 +986,12 @@
     )
     private int retentionCheckIntervalInSeconds = 120;
 
+    @Deprecated
     @FieldContext(
             category = CATEGORY_SERVER,
-            doc = "Check between intervals to see if max message size of topic policy has updated. default is 60s"
+            deprecated = true,
+            doc = "Check between intervals to see if  of topic policy has updated. default is 60s."
+                + "Note: This is deprecated, max message size is updated by notification."
     )
     private int maxMessageSizeCheckIntervalInSeconds = 60;
```

Comment:
I think we can remove it directly, it will not introduce any compatibility issues after users upgrade to the new version, and please also delete the config from the broker.conf and standalone.conf